packages/react-dom/src/client/ReactDOMFiberTextarea.js

```diff
@@ -126,7 +127,7 @@ export function updateWrapper(element: Element, props: Object) {
   if (value != null) {
     // Cast `value` to a string to ensure the value is set correctly. While
     // browsers typically do this as necessary, jsdom doesn't.
-    const newValue = '' + value;
+    const newValue = getSafeValue(value);
```

Comment:
I have a question about this change - could someone please be more specific about which cases require the casting done previously (`'' + value`) and what exactly breaks by removing it here?

Reply:
My first recollection is that we don't need to convert these to string values, but we didn't update all of the locations because it wasn't specific to the PR. 

Inputs went through a lot of fixes as we ironed out all of the edge cases for special input types on Chrome/Safari, so to some degree, touching more than we needed to felt scary 🙈. 

I think it would be good to take a second look at this.